src/components/Main/MyPresentation/index.jsx

```diff
@@ -37,9 +39,11 @@ function MyPresentation({ presentations }) {
               state={{ objects }}
             >
               <Thumbnail>
-                {thumbnailObjects.map((object) => (
-                  <DynamicObject key={object._id} objectSpec={object} />
-                ))}
+                {thumbnailObjects &&
+                  thumbnailObjects.length > 0 &&
+                  thumbnailObjects.map((object) => (
+                    <DynamicObject key={object._id} objectSpec={object} />
+                  ))}
```

Comment:
```suggestion
                {thumbnailObjects.length > 0 &&
                  thumbnailObjects.map((object) => (
                    <DynamicObject key={object._id} objectSpec={object} />
                  ))}
```
thumbnailObject의 기본값이 빈 배열이기 때문에 thumbnailObjects 자체는 항상 true일 것으로 보입니다. 이렇게 고쳐주세요 🙇‍♂️